### DISCUSSION TITLE
[Parseq example] Movement: pulsing forwards, rhythmic (translation z axis, incremented by sinusoidal amount relative to previous frame)

### DISCUSSION THREAD

**Discussion by rewbs** [rewbs/sd-parseq#118] [Parseq example library]
https://github.com/rewbs/sd-parseq/assets/74455/73e4a23e-e55b-4ebb-b4f2-c4eab0ecf2a2

Audio-sync'ed translation along the z axis by a sinusoidal increment relative to the previous frame using prev_computed_value and an exponentiated sine oscillator. Gives the impression that the viewer is "pulsing forwards".

🚀 [Parseq direct link](https://sd-parseq.web.app/?importRemote=GUWZvNjsIQOWlB5HVb2uT3vEg4u1%2Fdoc-1c05d528-a639-4d0e-896a-0d257cb9acc2-1685685964903.json&token=afcb6747-4c3d-46ed-b5fa-9c07cf50f473)
⬇️ [Parsec doc](https://gist.github.com/rewbs/1aefe8a4267714e17f8fdfbed2f23e61)
📃 [Deforum settings](https://gist.github.com/rewbs/9cc6c336f551e0c6517be8e43e44f679)
ℹ️ Post processing notes: e.g. x3 FILM (20fps to 60fps)

**Comment by shlomitgueta** [comment]
Thank you, I will try now
